README.md

```diff
@@ -116,14 +116,16 @@ We encourage all users of K8GB to add themselves to this list!
 
 ## Production Readiness
 
-k8gb is very well tested with the following environment options
+You can use k8gb in on-prem, cloud and hybrid environments.
+
+k8gb is tested with the following environment options.
 
 | Type                             | Implementation                                                               |
 |----------------------------------|------------------------------------------------------------------------------|
-| Kubernetes Version               | for k8s `< 1.19` use k8gb `<= 0.8.8`; since k8s `1.19` use `0.9.0` or newer  |
-| Environment                      | Self-managed, AWS(EKS) [*](#clarify)                                         |
+| Kubernetes Version               | >= `1.19`                                                                    |
+| Environment                      | On-prem(Self-managed K8S, Rancher), AWS(EKS), Azure(AKS) [*](#clarify)       |
```

Comment:
Is there anything on the k8gb installation that is specific to the Kubernetes distribution?
If not then we can simply write "Kubernetes" as the Environment

Reply:
yes, that make sense, let me update it